src/components/PlayerView/index.tsx

```diff
@@ -100,6 +105,7 @@ export function PlayerView({
   fullscreenHandler,
   customMessageHandler,
   isFullscreenRequested = false,
+  scalingMode = ScalingMode.Fit,
```

Comment:
Please remove the default value here.
Default values should be maintained in the native Player SDKs.

```suggestion
  scalingMode,
```